app/services/analytics_events.rb

```diff
@@ -126,6 +126,59 @@ def banned_user_visited
     track_event('Banned User visited')
   end
 
+  # @identity.idp.event_name Broken Personal Key: Regenerated
+  # A user that had a broken personal key was routed to a page to regenerate their personal key,
+  # so that they no longer have a broken one
+  def broken_personal_key_regenerated
+    track_event('Broken Personal Key: Regenerated')
+  end
+
+  # @identity.idp.event_name Doc Auth Async
+  # @param [String] error error message
+  # @param [String] uuid document capture session uuid
+  # @param [String] result_id document capture session result id
```

Comment:
Since `nil` can be a valid value, would we want to include it in the types?

```suggestion
  # @param [String, nil] error error message
  # @param [String, nil] uuid document capture session uuid
  # @param [String. nil] result_id document capture session result id
```

Reply:
My approach so far has been pretty lax on marking fields as nillable, so I'm inclined to leave as-is. How strongly do you feel?

Reply:
Hm, I think it could be valuable in having a distinction when we know a particular property may be a string or may be `nil`. It had caught my attention because of the [case below where we don't provide `result_id`](https://github.com/18F/identity-idp/blob/ca57cc9f53112214fdd860dc6887a56e1f33c7fd/app/services/idv/actions/verify_document_status_action.rb#L119-L122). 

Is the idea with being lax that we may consider any value as potentially nil-able?

Reply:
Exactly, I see almost any value as being nillable

But as long as we're here, doesn't hurt to make it a little more accurate, I guess. I'll take the change